timely_beliefs/beliefs/classes.py

```diff
@@ -667,7 +667,7 @@ def set_event_value_from_source(
     @property
     def knowledge_times(self) -> pd.DatetimeIndex:
         return pd.DatetimeIndex(
-            self.event_starts.to_series(name="knowledge_time").apply(
+            self.event_starts.to_series(name="knowledge_time", keep_tz=True).apply(
```

Comment:
These seem to be out of scope for this PR?

Reply:
Wrong base branch for merge.